{"asctime": "2020-04-22 17:46:56,012", "name": "core.message", "levelname": "INFO", "message": "this file has %d words", "run_duration": 0.9668381214141846}
{"asctime": "2020-04-22 17:49:25,729", "name": "core.message", "levelname": "ERROR", "message": "division by zero", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 35, in proccess\n    Y = (10 / 0)\nZeroDivisionError: division by zero"}
{"asctime": "2020-04-22 21:41:01,868", "name": "django.request", "levelname": "ERROR", "message": "Internal Server Error: /event/", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 34, in inner\n    response = get_response(request)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 115, in _get_response\n    response = self.process_exception_by_middleware(e, request)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 113, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/usr/lib/python3.8/contextlib.py\", line 74, in inner\n    with self._recreate_cm():\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/db/transaction.py\", line 175, in __enter__\n    if not connection.get_autocommit():\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/db/backends/base/base.py\", line 390, in get_autocommit\n    self.ensure_connection()\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/utils/asyncio.py\", line 26, in inner\n    return func(*args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/db/backends/base/base.py\", line 220, in ensure_connection\n    self.connect()\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/test/testcases.py\", line 144, in __call__\n    raise AssertionError(self.message)\nAssertionError: Database connections to 'app' are not allowed in this test. Add 'app' to core.tests.IndexViewTest.databases to ensure proper test isolation and silence this failure.", "status_code": 500, "request": "<WSGIRequest: GET '/event/'>"}
{"asctime": "2020-04-22 21:43:17,559", "name": "django.request", "levelname": "ERROR", "message": "Internal Server Error: /event/", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 34, in inner\n    response = get_response(request)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 115, in _get_response\n    response = self.process_exception_by_middleware(e, request)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 113, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/usr/lib/python3.8/contextlib.py\", line 74, in inner\n    with self._recreate_cm():\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/db/transaction.py\", line 175, in __enter__\n    if not connection.get_autocommit():\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/db/backends/base/base.py\", line 390, in get_autocommit\n    self.ensure_connection()\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/utils/asyncio.py\", line 26, in inner\n    return func(*args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/db/backends/base/base.py\", line 220, in ensure_connection\n    self.connect()\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/test/testcases.py\", line 144, in __call__\n    raise AssertionError(self.message)\nAssertionError: Database connections to 'app' are not allowed in this test. Add 'app' to core.tests.IndexViewTest.databases to ensure proper test isolation and silence this failure.", "status_code": 500, "request": "<WSGIRequest: GET '/event/'>"}
{"asctime": "2020-04-22 21:45:45,130", "name": "django.request", "levelname": "ERROR", "message": "Internal Server Error: /event/", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/core/handlers/exception.py\", line 34, in inner\n    response = get_response(request)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 115, in _get_response\n    response = self.process_exception_by_middleware(e, request)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/core/handlers/base.py\", line 113, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/usr/lib/python3.8/contextlib.py\", line 74, in inner\n    with self._recreate_cm():\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/db/transaction.py\", line 175, in __enter__\n    if not connection.get_autocommit():\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/db/backends/base/base.py\", line 390, in get_autocommit\n    self.ensure_connection()\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/utils/asyncio.py\", line 26, in inner\n    return func(*args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/db/backends/base/base.py\", line 220, in ensure_connection\n    self.connect()\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/test/testcases.py\", line 144, in __call__\n    raise AssertionError(self.message)\nAssertionError: Database connections to 'app' are not allowed in this test. Add 'app' to core.tests.IndexViewTest.databases to ensure proper test isolation and silence this failure.", "status_code": 500, "request": "<WSGIRequest: GET '/event/'>"}
{"asctime": "2020-04-23 12:34:15,701", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.5672738552093506}
{"asctime": "2020-04-23 12:34:30,725", "name": "core.message", "levelname": "ERROR", "message": "expected str, bytes or os.PathLike object, not _TemporaryFileWrapper", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 37, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 231, in callback_graph\n    msg_photo(pyplot, msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 238, in msg_photo\n    bot.sendPhoto(chat_id=msg['user_id'], photo=open(img, 'rb'))\nTypeError: expected str, bytes or os.PathLike object, not _TemporaryFileWrapper"}
{"asctime": "2020-04-23 12:36:25,120", "name": "core.message", "levelname": "ERROR", "message": "expected str, bytes or os.PathLike object, not _TemporaryFileWrapper", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 37, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 231, in callback_graph\n    msg_photo(pyplot, msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 238, in msg_photo\n    bot.sendPhoto(chat_id=msg['user_id'], photo=open(img, 'rb'))\nTypeError: expected str, bytes or os.PathLike object, not _TemporaryFileWrapper"}
{"asctime": "2020-04-23 12:37:37,954", "name": "core.message", "levelname": "ERROR", "message": "expected str, bytes or os.PathLike object, not _TemporaryFileWrapper", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 37, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 231, in callback_graph\n    msg_photo(pyplot, msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 238, in msg_photo\n    bot.sendPhoto(chat_id=msg['user_id'], photo=open(img, 'rb'))\nTypeError: expected str, bytes or os.PathLike object, not _TemporaryFileWrapper"}
{"asctime": "2020-04-23 12:39:32,049", "name": "core.message", "levelname": "ERROR", "message": "Image_process_failed", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 37, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 234, in callback_graph\n    bot.sendPhoto(chat_id=msg['user_id'], photo=open(img.name, 'rb'))\n  File \"<decorator-gen-5>\", line 2, in send_photo\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 70, in decorator\n    result = func(*args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 498, in send_photo\n    return self._message(url, data, timeout=timeout, disable_notification=disable_notification,\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 178, in _message\n    result = self._request.post(url, data, timeout=timeout)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/utils/request.py\", line 329, in post\n    result = self._request_wrapper('POST', url, fields=data, **urlopen_kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/utils/request.py\", line 245, in _request_wrapper\n    raise BadRequest(message)\ntelegram.error.BadRequest: Image_process_failed"}
{"asctime": "2020-04-23 12:40:47,983", "name": "core.message", "levelname": "ERROR", "message": "Image_process_failed", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 37, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 234, in callback_graph\n    bot.sendPhoto(chat_id=msg['user_id'], photo=open(img.name, 'rb'))\n  File \"<decorator-gen-5>\", line 2, in send_photo\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 70, in decorator\n    result = func(*args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 498, in send_photo\n    return self._message(url, data, timeout=timeout, disable_notification=disable_notification,\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 178, in _message\n    result = self._request.post(url, data, timeout=timeout)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/utils/request.py\", line 329, in post\n    result = self._request_wrapper('POST', url, fields=data, **urlopen_kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/utils/request.py\", line 245, in _request_wrapper\n    raise BadRequest(message)\ntelegram.error.BadRequest: Image_process_failed"}
{"asctime": "2020-04-23 12:43:46,773", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.10215163230896}
{"asctime": "2020-04-23 12:44:49,953", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.1057939529418945}
{"asctime": "2020-04-23 12:45:07,662", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.883815050125122}
{"asctime": "2020-04-23 12:50:15,477", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.1813616752624512}
{"asctime": "2020-04-23 12:50:21,634", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.28765344619751}
{"asctime": "2020-04-23 12:50:32,853", "name": "core.message", "levelname": "ERROR", "message": "expected str, bytes or os.PathLike object, not _TemporaryFileWrapper", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 42, in proccess\n    callback_spread(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 259, in callback_spread\n    bot.sendPhoto(chat_id=msg['user_id'], photo=open(img, 'rb'))\nTypeError: expected str, bytes or os.PathLike object, not _TemporaryFileWrapper"}
{"asctime": "2020-04-23 12:51:01,362", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 0.5778849124908447}
{"asctime": "2020-04-23 12:51:08,119", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.000624179840088}
{"asctime": "2020-04-23 12:51:13,799", "name": "core.message", "levelname": "ERROR", "message": "expected str, bytes or os.PathLike object, not _TemporaryFileWrapper", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 42, in proccess\n    callback_spread(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 259, in callback_spread\n    bot.sendPhoto(chat_id=msg['user_id'], photo=open(img, 'rb'))\nTypeError: expected str, bytes or os.PathLike object, not _TemporaryFileWrapper"}
{"asctime": "2020-04-23 12:52:32,593", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.5942628383636475}
{"asctime": "2020-04-23 12:52:38,881", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.002449035644531}
{"asctime": "2020-04-23 12:52:43,716", "name": "core.message", "levelname": "ERROR", "message": "File must be non-empty", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 42, in proccess\n    callback_spread(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 260, in callback_spread\n    bot.sendPhoto(chat_id=msg['user_id'], photo=open(img, 'rb'))\n  File \"<decorator-gen-5>\", line 2, in send_photo\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 70, in decorator\n    result = func(*args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 498, in send_photo\n    return self._message(url, data, timeout=timeout, disable_notification=disable_notification,\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 178, in _message\n    result = self._request.post(url, data, timeout=timeout)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/utils/request.py\", line 329, in post\n    result = self._request_wrapper('POST', url, fields=data, **urlopen_kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/utils/request.py\", line 245, in _request_wrapper\n    raise BadRequest(message)\ntelegram.error.BadRequest: File must be non-empty"}
{"asctime": "2020-04-23 12:54:18,582", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.0217955112457275}
{"asctime": "2020-04-23 12:54:23,962", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 3.910090684890747}
{"asctime": "2020-04-23 12:54:29,453", "name": "core.message", "levelname": "ERROR", "message": "Image_process_failed", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 42, in proccess\n    callback_spread(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 259, in callback_spread\n    bot.sendPhoto(chat_id=msg['user_id'], photo=open(img.name, 'rb'))\n  File \"<decorator-gen-5>\", line 2, in send_photo\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 70, in decorator\n    result = func(*args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 498, in send_photo\n    return self._message(url, data, timeout=timeout, disable_notification=disable_notification,\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 178, in _message\n    result = self._request.post(url, data, timeout=timeout)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/utils/request.py\", line 329, in post\n    result = self._request_wrapper('POST', url, fields=data, **urlopen_kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/utils/request.py\", line 245, in _request_wrapper\n    raise BadRequest(message)\ntelegram.error.BadRequest: Image_process_failed"}
{"asctime": "2020-04-23 12:56:45,636", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.176229476928711}
{"asctime": "2020-04-23 12:56:50,984", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.025302410125732}
{"asctime": "2020-04-23 12:56:57,814", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.02203893661499}
{"asctime": "2020-04-23 12:58:46,571", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.3899013996124268}
{"asctime": "2020-04-23 12:58:52,209", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 3.907238483428955}
{"asctime": "2020-04-23 12:58:59,371", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.906326532363892}
{"asctime": "2020-04-23 12:59:58,453", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 0.9784307479858398}
{"asctime": "2020-04-23 13:00:03,984", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.001898527145386}
{"asctime": "2020-04-23 13:00:11,768", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.335531711578369}
{"asctime": "2020-04-23 13:01:33,274", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 0.4112434387207031}
{"asctime": "2020-04-23 13:01:41,218", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 3.9968807697296143}
{"asctime": "2020-04-23 13:01:57,815", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 0.9975337982177734}
{"asctime": "2020-04-23 13:02:02,973", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 3.811847448348999}
{"asctime": "2020-04-23 13:02:10,967", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.730097770690918}
{"asctime": "2020-04-23 13:03:42,735", "name": "core.message", "levelname": "ERROR", "message": "File must be non-empty", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 37, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 236, in callback_graph\n    bot.sendPhoto(chat_id=msg['user_id'], photo=open(img, 'rb'))\n  File \"<decorator-gen-5>\", line 2, in send_photo\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 70, in decorator\n    result = func(*args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 498, in send_photo\n    return self._message(url, data, timeout=timeout, disable_notification=disable_notification,\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/bot.py\", line 178, in _message\n    result = self._request.post(url, data, timeout=timeout)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/utils/request.py\", line 329, in post\n    result = self._request_wrapper('POST', url, fields=data, **urlopen_kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/telegram/utils/request.py\", line 245, in _request_wrapper\n    raise BadRequest(message)\ntelegram.error.BadRequest: File must be non-empty"}
{"asctime": "2020-04-23 13:04:46,928", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.466410875320435}
{"asctime": "2020-04-23 13:04:53,699", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.037186145782471}
{"asctime": "2020-04-23 13:06:07,823", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.170374631881714}
{"asctime": "2020-04-23 13:07:26,371", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.226618766784668}
{"asctime": "2020-04-23 13:07:43,829", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.030592203140259}
{"asctime": "2020-04-23 13:09:17,682", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.231964349746704}
{"asctime": "2020-04-23 13:09:23,392", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.057883977890015}
{"asctime": "2020-04-23 13:10:08,775", "name": "core.message", "levelname": "ERROR", "message": "name 'img' is not defined", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 42, in proccess\n    callback_spread(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 268, in callback_spread\n    os.unlink(img)\nNameError: name 'img' is not defined"}
{"asctime": "2020-04-23 13:14:51,261", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.1818602085113525}
{"asctime": "2020-04-23 13:14:58,019", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.03413987159729}
{"asctime": "2020-04-23 13:15:04,415", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.935932874679565}
{"asctime": "2020-04-23 13:22:57,871", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.0612657070159912}
{"asctime": "2020-04-23 13:23:08,369", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.047002792358398}
{"asctime": "2020-04-23 13:23:15,848", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.741840362548828}
{"asctime": "2020-04-23 14:00:25,812", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 8.71329927444458}
{"asctime": "2020-04-23 14:01:52,075", "name": "core.message", "levelname": "ERROR", "message": "name 'plt' is not defined", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 37, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 103, in callback_graph\n    plt.figtext(0, 99, 0, 1, 'texto da nota de rodap\u00e9', horizontalalignment='right')\nNameError: name 'plt' is not defined"}
{"asctime": "2020-04-23 14:02:14,320", "name": "core.message", "levelname": "ERROR", "message": "'int' object has no attribute 'pop'", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 37, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 103, in callback_graph\n    pyplot.figtext(0, 99, 0, 1, 'texto da nota de rodap\u00e9', horizontalalignment='right')\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/pyplot.py\", line 2310, in figtext\n    return gcf().text(\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/cbook/deprecation.py\", line 358, in wrapper\n    return func(*args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/figure.py\", line 1896, in text\n    text.update(fontdict)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/text.py\", line 176, in update\n    bbox = kwargs.pop(\"bbox\", sentinel)\nAttributeError: 'int' object has no attribute 'pop'"}
{"asctime": "2020-04-23 14:05:07,359", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.2198216915130615}
{"asctime": "2020-04-23 14:13:42,733", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.424511671066284}
{"asctime": "2020-04-23 14:15:08,450", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 8.57603645324707}
{"asctime": "2020-04-23 14:16:20,220", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.049147844314575}
{"asctime": "2020-04-23 14:16:48,378", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.876323699951172}
{"asctime": "2020-04-23 14:17:10,190", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.021553039550781}
{"asctime": "2020-04-23 14:40:44,185", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.274569034576416}
{"asctime": "2020-04-23 14:40:58,566", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 11.626170635223389}
{"asctime": "2020-04-23 14:41:41,742", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 14:42:41,887", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 14.031461477279663}
{"asctime": "2020-04-23 14:43:38,758", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.768109560012817}
{"asctime": "2020-04-23 14:44:47,214", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.4733710289001465}
{"asctime": "2020-04-23 14:45:28,779", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 14:47:20,702", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.74958610534668}
{"asctime": "2020-04-23 14:48:11,588", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.186073541641235}
{"asctime": "2020-04-23 14:49:11,801", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.037832021713257}
{"asctime": "2020-04-23 14:49:46,200", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.114139556884766}
{"asctime": "2020-04-23 14:50:04,944", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.7684104442596436}
{"asctime": "2020-04-23 14:50:39,839", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 14:51:49,092", "name": "core.message", "levelname": "ERROR", "message": "name 'ax' is not defined", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 37, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 111, in callback_graph\n    transform=ax.transAxes,\nNameError: name 'ax' is not defined"}
{"asctime": "2020-04-23 14:52:35,159", "name": "core.message", "levelname": "ERROR", "message": "module 'matplotlib.pyplot' has no attribute 'transAxes'", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 37, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 111, in callback_graph\n    transform=pyplot.transAxes,\nAttributeError: module 'matplotlib.pyplot' has no attribute 'transAxes'"}
{"asctime": "2020-04-23 14:53:11,044", "name": "core.message", "levelname": "ERROR", "message": "'function' object has no attribute 'transform'", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 37, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 119, in callback_graph\n    msg_photo(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 289, in msg_photo\n    pyplot.savefig(img, bbox_inches='tight', format='png')\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/pyplot.py\", line 723, in savefig\n    res = fig.savefig(*args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/figure.py\", line 2203, in savefig\n    self.canvas.print_figure(fname, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/backend_bases.py\", line 2079, in print_figure\n    self.figure.draw(renderer)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/artist.py\", line 38, in draw_wrapper\n    return draw(artist, renderer, *args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/figure.py\", line 1735, in draw\n    mimage._draw_list_compositing_images(\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/image.py\", line 137, in _draw_list_compositing_images\n    a.draw(renderer)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/artist.py\", line 38, in draw_wrapper\n    return draw(artist, renderer, *args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/axes/_base.py\", line 2630, in draw\n    mimage._draw_list_compositing_images(renderer, self, artists)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/image.py\", line 137, in _draw_list_compositing_images\n    a.draw(renderer)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/artist.py\", line 38, in draw_wrapper\n    return draw(artist, renderer, *args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/text.py\", line 692, in draw\n    posx, posy = trans.transform((posx, posy))\nAttributeError: 'function' object has no attribute 'transform'"}
{"asctime": "2020-04-23 14:53:35,984", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.339223146438599}
{"asctime": "2020-04-23 14:54:18,171", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.018458843231201}
{"asctime": "2020-04-23 14:55:09,222", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.870582818984985}
{"asctime": "2020-04-23 14:59:07,141", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 14:59:53,619", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.525648593902588}
{"asctime": "2020-04-23 15:02:01,701", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 15:04:45,990", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 9.357293367385864}
{"asctime": "2020-04-23 15:24:26,457", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.6281585693359375}
{"asctime": "2020-04-23 15:26:04,565", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.17651629447937}
{"asctime": "2020-04-23 15:27:48,314", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 15:29:54,716", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 15:31:46,475", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.316669940948486}
{"asctime": "2020-04-23 15:32:22,629", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.706537246704102}
{"asctime": "2020-04-23 16:53:04,268", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 16:55:14,225", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 16:57:14,723", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 16:57:58,722", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 17:08:26,542", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.2485013008117676}
{"asctime": "2020-04-23 17:08:39,431", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 8.505073547363281}
{"asctime": "2020-04-23 17:10:25,631", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 12.512558460235596}
{"asctime": "2020-04-23 17:10:30,224", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.1861348152160645}
{"asctime": "2020-04-23 17:11:07,188", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.513715028762817}
{"asctime": "2020-04-23 17:11:47,164", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.012990713119507}
{"asctime": "2020-04-23 17:13:08,939", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.27856183052063}
{"asctime": "2020-04-23 17:19:26,205", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 17:21:58,582", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 8.062317132949829}
{"asctime": "2020-04-23 17:23:20,266", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.0851705074310303}
{"asctime": "2020-04-23 17:23:25,638", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.259398698806763}
{"asctime": "2020-04-23 17:25:11,084", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 17:29:35,786", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 8.85897183418274}
{"asctime": "2020-04-23 17:30:40,467", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.853971481323242}
{"asctime": "2020-04-23 17:31:05,565", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.023276329040527}
{"asctime": "2020-04-23 17:31:41,400", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.173276662826538}
{"asctime": "2020-04-23 17:32:34,555", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.412947416305542}
{"asctime": "2020-04-23 17:33:03,842", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.940954685211182}
{"asctime": "2020-04-23 17:33:36,609", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.381489038467407}
{"asctime": "2020-04-23 17:34:05,396", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.729668140411377}
{"asctime": "2020-04-23 17:35:14,449", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.022691488265991}
{"asctime": "2020-04-23 17:36:02,919", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.776200294494629}
{"asctime": "2020-04-23 17:36:35,602", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.42283034324646}
{"asctime": "2020-04-23 17:37:07,403", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.2668182849884033}
{"asctime": "2020-04-23 17:37:13,291", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.411950588226318}
{"asctime": "2020-04-23 17:37:50,474", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.366632461547852}
{"asctime": "2020-04-23 17:38:25,945", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.360732078552246}
{"asctime": "2020-04-23 17:45:35,707", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 17:49:46,682", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 9.012072086334229}
{"asctime": "2020-04-23 17:51:14,429", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.795301675796509}
{"asctime": "2020-04-23 17:52:11,158", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.693929433822632}
{"asctime": "2020-04-23 17:52:40,644", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.461962699890137}
{"asctime": "2020-04-23 17:55:22,352", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 17:55:53,157", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 17:56:18,335", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 17:56:55,723", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 8.224185705184937}
{"asctime": "2020-04-23 17:57:56,849", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.139386892318726}
{"asctime": "2020-04-23 17:58:44,479", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.3316357135772705}
{"asctime": "2020-04-23 18:06:22,617", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 18:06:58,052", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.581356763839722}
{"asctime": "2020-04-23 18:07:59,183", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.554525375366211}
{"asctime": "2020-04-23 18:09:28,067", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.130092144012451}
{"asctime": "2020-04-23 18:11:44,117", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 0.9798758029937744}
{"asctime": "2020-04-23 18:11:50,159", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.667463302612305}
{"asctime": "2020-04-23 18:13:46,471", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.9488301277160645}
{"asctime": "2020-04-23 18:14:43,943", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.505829811096191}
{"asctime": "2020-04-23 18:16:03,385", "name": "core.message", "levelname": "ERROR", "message": "__init__() got an unexpected keyword argument 'bbox'", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 38, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 120, in callback_graph\n    pyplot.legend(bbox=(0., 1.02, 1., .102), loc='lower left', ncol=2)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/pyplot.py\", line 2659, in legend\n    return gca().legend(*args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/axes/_axes.py\", line 401, in legend\n    self.legend_ = mlegend.Legend(self, handles, labels, **kwargs)\nTypeError: __init__() got an unexpected keyword argument 'bbox'"}
{"asctime": "2020-04-23 18:17:53,637", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.2534544467926025}
{"asctime": "2020-04-23 18:20:35,371", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.46187424659729}
{"asctime": "2020-04-23 18:20:58,996", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.1351001262664795}
{"asctime": "2020-04-23 18:25:07,119", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 18:29:13,437", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.7814624309539795}
{"asctime": "2020-04-23 18:30:04,776", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.214655637741089}
{"asctime": "2020-04-23 18:31:02,590", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.2277841567993164}
{"asctime": "2020-04-23 18:31:10,220", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.311425447463989}
{"asctime": "2020-04-23 18:33:31,214", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.377650737762451}
{"asctime": "2020-04-23 18:33:54,343", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 18:34:47,140", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.304764032363892}
{"asctime": "2020-04-23 18:36:04,764", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 18:37:02,813", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.760625839233398}
{"asctime": "2020-04-23 18:37:23,396", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.2216572761535645}
{"asctime": "2020-04-23 18:37:49,375", "name": "core.message", "levelname": "ERROR", "message": "'str' object has no attribute 'pop'", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 38, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 107, in callback_graph\n    pyplot.figtext(0.99, 0,9, 'Concession\u00e1ria', horizontalalignment='right')\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/pyplot.py\", line 2310, in figtext\n    return gcf().text(\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/cbook/deprecation.py\", line 358, in wrapper\n    return func(*args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/figure.py\", line 1896, in text\n    text.update(fontdict)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/text.py\", line 176, in update\n    bbox = kwargs.pop(\"bbox\", sentinel)\nAttributeError: 'str' object has no attribute 'pop'"}
{"asctime": "2020-04-23 18:38:14,184", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.692075490951538}
{"asctime": "2020-04-23 18:38:58,413", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.375659465789795}
{"asctime": "2020-04-23 18:43:05,221", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 18:45:41,864", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 18:50:22,703", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 18:51:58,842", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 18:54:39,282", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 18:55:32,234", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 18:55:52,110", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 18:56:05,524", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 18:56:29,569", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 18:57:27,256", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 8.86025619506836}
{"asctime": "2020-04-23 18:57:52,007", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.561122894287109}
{"asctime": "2020-04-23 19:05:47,286", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.5151777267456055}
{"asctime": "2020-04-23 19:07:00,976", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.489546537399292}
{"asctime": "2020-04-23 19:10:55,512", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 19:11:48,797", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 19:14:09,689", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 19:15:05,949", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 9.638591289520264}
{"asctime": "2020-04-23 19:18:15,555", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 8.39597225189209}
{"asctime": "2020-04-23 19:31:04,565", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.2914113998413086}
{"asctime": "2020-04-23 19:31:14,026", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 8.155348539352417}
{"asctime": "2020-04-23 19:34:46,751", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 19:37:39,663", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 8.588994264602661}
{"asctime": "2020-04-23 19:38:21,963", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.42036247253418}
{"asctime": "2020-04-23 19:38:49,794", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.9529430866241455}
{"asctime": "2020-04-23 19:39:16,990", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.884778261184692}
{"asctime": "2020-04-23 19:39:41,508", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.538341999053955}
{"asctime": "2020-04-23 19:39:58,056", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.211074113845825}
{"asctime": "2020-04-23 19:40:13,730", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.728992700576782}
{"asctime": "2020-04-23 19:40:13,922", "name": "telegram.vendor.ptb_urllib3.urllib3.connectionpool", "levelname": "WARNING", "message": "Connection pool is full, discarding connection: api.telegram.org"}
{"asctime": "2020-04-23 19:40:13,926", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.107537269592285}
{"asctime": "2020-04-23 19:40:51,609", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.861123561859131}
{"asctime": "2020-04-23 19:41:07,325", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.450967311859131}
{"asctime": "2020-04-23 19:41:08,450", "name": "telegram.vendor.ptb_urllib3.urllib3.connectionpool", "levelname": "WARNING", "message": "Connection pool is full, discarding connection: api.telegram.org"}
{"asctime": "2020-04-23 19:41:09,544", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.4974987506866455}
{"asctime": "2020-04-23 19:41:27,653", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.728322744369507}
{"asctime": "2020-04-23 19:41:46,186", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.69643235206604}
{"asctime": "2020-04-23 19:41:46,392", "name": "telegram.vendor.ptb_urllib3.urllib3.connectionpool", "levelname": "WARNING", "message": "Connection pool is full, discarding connection: api.telegram.org"}
{"asctime": "2020-04-23 19:41:46,398", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.902288436889648}
{"asctime": "2020-04-23 19:42:01,732", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.109619855880737}
{"asctime": "2020-04-23 19:42:40,151", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 7.771152019500732}
{"asctime": "2020-04-23 19:47:45,661", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.2491552829742432}
{"asctime": "2020-04-23 19:47:57,179", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.116908073425293}
{"asctime": "2020-04-23 19:48:29,053", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.251561164855957}
{"asctime": "2020-04-23 19:48:56,512", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.798971652984619}
{"asctime": "2020-04-23 19:49:36,816", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.403480768203735}
{"asctime": "2020-04-23 19:50:05,801", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.290738344192505}
{"asctime": "2020-04-23 19:50:38,052", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.041530132293701}
{"asctime": "2020-04-23 19:50:57,410", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.612009286880493}
{"asctime": "2020-04-23 19:51:28,050", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 0.9724197387695312}
{"asctime": "2020-04-23 19:51:34,496", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.958400011062622}
{"asctime": "2020-04-23 19:51:53,418", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.345007658004761}
{"asctime": "2020-04-23 19:52:16,522", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 7.074361801147461}
{"asctime": "2020-04-23 19:53:08,783", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 19:57:20,111", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 9.080323696136475}
{"asctime": "2020-04-23 19:57:43,743", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 20:00:24,334", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 20:01:41,453", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 14.183231830596924}
{"asctime": "2020-04-23 20:02:22,221", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 20:02:59,489", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 7.475738525390625}
{"asctime": "2020-04-23 20:03:53,818", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.405400276184082}
{"asctime": "2020-04-23 20:04:40,030", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.0077874660491943}
{"asctime": "2020-04-23 20:04:46,451", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.948673248291016}
{"asctime": "2020-04-23 20:05:11,134", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.101165533065796}
{"asctime": "2020-04-23 20:06:07,198", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.693833112716675}
{"asctime": "2020-04-23 20:06:23,737", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.2390305995941162}
{"asctime": "2020-04-23 20:06:31,725", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.637129545211792}
{"asctime": "2020-04-23 20:14:28,326", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 0.49027419090270996}
{"asctime": "2020-04-23 20:15:29,717", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 0.9923334121704102}
{"asctime": "2020-04-23 21:39:24,864", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-23 21:49:17,963", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 18.421685934066772}
{"asctime": "2020-04-23 21:49:55,304", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 0.4770514965057373}
{"asctime": "2020-04-23 21:50:05,516", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 8.126065492630005}
{"asctime": "2020-04-23 21:50:12,224", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.211613893508911}
{"asctime": "2020-04-24 12:09:58,438", "name": "core.message", "levelname": "ERROR", "message": "0", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 38, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 91, in callback_graph\n    xaxis.append('{:02d}:{:02d}'.format(row[0].hour, row[0].minute))\nKeyError: 0"}
{"asctime": "2020-04-24 12:10:00,459", "name": "core.message", "levelname": "ERROR", "message": "0", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 38, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 91, in callback_graph\n    xaxis.append('{:02d}:{:02d}'.format(row[0].hour, row[0].minute))\nKeyError: 0"}
{"asctime": "2020-04-24 12:12:08,906", "name": "core.message", "levelname": "ERROR", "message": "0", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 38, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 91, in callback_graph\n    xaxis.append('{:02d}:{:02d}'.format(row[0].hour, row[0].minute))\nKeyError: 0"}
{"asctime": "2020-04-24 14:59:32,056", "name": "core.message", "levelname": "ERROR", "message": "'psycopg2.extensions.cursor' object has no attribute 'dictfetchall'", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 38, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 86, in callback_graph\n    dic = get_data(code)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 154, in get_data\n    data = cur.dictfetchall()()\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/db/backends/utils.py\", line 24, in __getattr__\n    cursor_attr = getattr(self.cursor, attr)\nAttributeError: 'psycopg2.extensions.cursor' object has no attribute 'dictfetchall'"}
{"asctime": "2020-04-24 15:04:12,097", "name": "core.message", "levelname": "ERROR", "message": "cursor() got an unexpected keyword argument 'cursor_factory'", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 40, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 88, in callback_graph\n    dic = get_data(code)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 155, in get_data\n    cur = connections['app'].cursor(cursor_factory=psycopg2.extras.DictCursor)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/utils/asyncio.py\", line 26, in inner\n    return func(*args, **kwargs)\nTypeError: cursor() got an unexpected keyword argument 'cursor_factory'"}
{"asctime": "2020-04-24 15:08:39,228", "name": "core.message", "levelname": "ERROR", "message": "cursor() got an unexpected keyword argument 'cursor_factory'", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 40, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 88, in callback_graph\n    dic = get_data(code)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 155, in get_data\n    cur = connections['app'].cursor(cursor_factory=psycopg2.extras.DictCursor)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/utils/asyncio.py\", line 26, in inner\n    return func(*args, **kwargs)\nTypeError: cursor() got an unexpected keyword argument 'cursor_factory'"}
{"asctime": "2020-04-24 15:10:41,014", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-24 15:11:42,836", "name": "core.message", "levelname": "ERROR", "message": "cursor() got an unexpected keyword argument 'cursor_factory'", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 40, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 88, in callback_graph\n    dic = get_data(code)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 155, in get_data\n    cur = connections['app'].cursor(cursor_factory=NamedTupleCursor)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/utils/asyncio.py\", line 26, in inner\n    return func(*args, **kwargs)\nTypeError: cursor() got an unexpected keyword argument 'cursor_factory'"}
{"asctime": "2020-04-24 17:41:51,446", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-24 17:48:49,064", "name": "core.message", "levelname": "ERROR", "message": "'psycopg2.extensions.cursor' object has no attribute 'dictfetchall'", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 40, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 88, in callback_graph\n    dic = get_data(code)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 156, in get_data\n    data = cur.dictfetchall()()\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/django/db/backends/utils.py\", line 24, in __getattr__\n    cursor_attr = getattr(self.cursor, attr)\nAttributeError: 'psycopg2.extensions.cursor' object has no attribute 'dictfetchall'"}
{"asctime": "2020-04-24 17:49:44,932", "name": "core.message", "levelname": "ERROR", "message": "1", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 40, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 94, in callback_graph\n    yaxis.append(row[1])\nKeyError: 1"}
{"asctime": "2020-04-24 17:52:31,265", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 8.18536114692688}
{"asctime": "2020-04-24 17:53:09,260", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.0610461235046387}
{"asctime": "2020-04-24 17:53:17,208", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 4.840196132659912}
{"asctime": "2020-04-24 17:54:35,997", "name": "core.message", "levelname": "ERROR", "message": "'Text' object has no property 'fonztsize'", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 40, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 112, in callback_graph\n    pyplot.figtext(0.9, 0.9, dic[0]['equipment_name'], horizontalalignment='right', fonztsize=10)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/pyplot.py\", line 2310, in figtext\n    return gcf().text(\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/cbook/deprecation.py\", line 358, in wrapper\n    return func(*args, **kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/figure.py\", line 1897, in text\n    text.update(kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/text.py\", line 177, in update\n    super().update(kwargs)\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/artist.py\", line 1006, in update\n    ret = [_update_property(self, k, v) for k, v in props.items()]\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/artist.py\", line 1006, in <listcomp>\n    ret = [_update_property(self, k, v) for k, v in props.items()]\n  File \"/home/antonio/Documents/estudos/bot2/venv/lib/python3.8/site-packages/matplotlib/artist.py\", line 1001, in _update_property\n    raise AttributeError('{!r} object has no property {!r}'\nAttributeError: 'Text' object has no property 'fonztsize'"}
{"asctime": "2020-04-24 17:55:12,839", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 0.9158225059509277}
{"asctime": "2020-04-24 17:55:19,158", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 3.7946701049804688}
{"asctime": "2020-04-24 17:59:13,637", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.1846182346343994}
{"asctime": "2020-04-24 17:59:20,697", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 5.071470499038696}
{"asctime": "2020-04-24 18:02:23,445", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/models.py changed, reloading."}
{"asctime": "2020-04-24 18:08:55,838", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/models.py changed, reloading."}
{"asctime": "2020-04-24 18:09:51,708", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/models.py changed, reloading."}
{"asctime": "2020-04-24 18:42:41,222", "name": "django.utils.autoreload", "levelname": "INFO", "message": "/home/antonio/Documents/estudos/bot2/core/message.py changed, reloading."}
{"asctime": "2020-04-24 18:45:50,032", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.210085153579712}
{"asctime": "2020-04-24 18:45:59,107", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 6.574816465377808}
{"asctime": "2020-04-24 18:55:32,780", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 1.6033964157104492}
{"asctime": "2020-04-24 18:55:37,835", "name": "core.message", "levelname": "ERROR", "message": "Interaction Type n\u00e3o previsto: Column", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 102, in callback_graph\n    raise ValueError('Interaction Type n\u00e3o previsto: {}'.format(interaction.type))\nValueError: Interaction Type n\u00e3o previsto: Column"}
{"asctime": "2020-04-24 18:55:37,836", "name": "core.message", "levelname": "ERROR", "message": "Interaction Type n\u00e3o previsto: Column", "exc_info": "Traceback (most recent call last):\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 40, in proccess\n    callback_graph(msg)\n  File \"/home/antonio/Documents/estudos/bot2/core/message.py\", line 102, in callback_graph\n    raise ValueError('Interaction Type n\u00e3o previsto: {}'.format(interaction.type))\nValueError: Interaction Type n\u00e3o previsto: Column"}
{"asctime": "2020-04-24 18:55:54,185", "name": "core.message", "levelname": "INFO", "message": "Processando mensagem:", "run_duration": 0.3716614246368408}
